src/components/SmallButton.js

```diff
@@ -1,16 +1,17 @@
-import PropTypes from 'prop-types'
+import PropTypes, { string } from 'prop-types'
 import styled from 'styled-components/macro'
 
 SmallButton.propTypes = {
   onClick: PropTypes.func,
   children: PropTypes.node,
   top: PropTypes.string,
   right: PropTypes.string,
+  color: string,
```

Comment:
Is there a difference between PropTypes.string & string (You import {string} as well)? If not I would suggest to unify the look by using only PropTypes.* or the destructured statements

Reply:
I just forgot the `PropTypes`and apparently there was an auto import 😄